src/tests/ftest/util/test_utils_pool.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, context, dmg_command, cb_handler=None):
         self.pool_query_timeout = BasicParameter(None)
 
         self.pool = None
+        self.label = None
```

Comment:
Require an additional `label_generator` input to the `__init__()` method that is a new LabelGenerator object type:
```suggestion
        self.label = BasicParameter(None, "daos_pool")
        self.label_generator = label_generator
```

The `LabelGenerator` class would look something like this:
```
class LabelGenerator():

    def __init__(self, value=1):
        self.value = value

    def get_label(self, base_label):
        label = base_label
        if label is not None:
            label = "_".join([base_label, str(self.value)])
            self.value += 1
        return label
```